tests/device_tests/recovery/test_bip39.py

```diff
@@ -0,0 +1,215 @@
+# This file is part of the Trezor project.
```

Comment:
if this file is `skip_t2` as a whole, and we have a corresponding `_t2` variant, please also rename it to `test_bip39_t1`

there might be other cases that i'm not seeing now. If you know of similar others (i.e., (a) whole file is model-specific and (b) there is a corresponding file for the other model), we might want to rename them too

Reply:
Couple files found and renamed in d15ee5c